src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const listURL
+= `https://mate-academy.github.io/phone-catalogue-static/api/phones.json`;
+
+const detailsURL
+= `https://mate-academy.github.io/phone-catalogue-static/api/phones/`;
+
+const newList = document.createElement('ul');
+
+document.body.insertAdjacentElement('beforeend', newList);
+
+function getPhones() {
+  return fetch(listURL)
+    .then(response => response.json());
+}
+
+function getPhonesDetails(id) {
```

Comment:
```suggestion
function getPhoneDetails(id) {
```
Это же детали одного телефона